python/paddle/v2/parameters.py

```diff
@@ -187,7 +184,7 @@ def __setitem__(self, key, value):
                              (shape, value.shape))
 
         if len(self.__gradient_machines__) == 0:
-            self.__tmp_params__.append((key, value))
```

Comment:
The `append` could lead the repeated key in `self.__tmp_params__`. This is a bug.